Sources/ComposableArchitecture/Internal/Deprecations.swift

```diff
@@ -548,8 +544,7 @@ extension Reducer {
       }
       if index >= globalState[keyPath: toLocalState].endIndex {
         #if DEBUG
-          os_log(
-            .fault, dso: rw.dso, log: rw.log,
+          runtimeWarning(
```

Comment:
Moved things to a helper function, which means the purple warning unfortunately happens there now. It's not super helpful to highlight even here, though.

Reply:
You could use [@_transparent](https://github.com/apple/swift/blob/main/docs/TransparentAttr.md) if you wanted to get the warning back to the `runtimeWarning` call site

Reply:
Oh great! TIL :smile:

You up to PR that?

Reply:
Yeah no worries